docs/src/simulation_tutorial.md

```diff
@@ -0,0 +1,673 @@
+power_table(kb07_sim_half)
+```
+
+# 3. Create a (simple) fully crossed dataset from scratch and analyze power.
+
+In some situations, instead of using an existing dataset it may be useful to simulate the data from scratch. This could be the case when the original data is not available but the effect sizes are known. That means that we have to:
+
+a) specify the effect sizes manually
+b) manually create an experimental design, according to which data can be simulated
+
+If we simulate data from scratch, aside from subject and item number, we can manipulate the arguments `β`, `σ` and `θ`.
+Lets have a closer look at them, define their meaning and we will see where the corresponding values in the model output are.
+
+### **Beta**
+`β` are our effect sizes. If we look again on our LMM summary from the `kb07`-dataset `kb07_m`
+we see our four `β` under fixed-effects parameters in the `Coef.`-column.
+
+```@example Main
+kb07_m
+kb07_m.β
+```
+
+### **Sigma**
+`σ` is the residual-standard deviation listed under the variance components.
+```@example Main
+kb07_m
+kb07_m.σ
+```
+
+### **Theta**
+The meaning of `θ` is a bit less intuitive. In a less complex model (one that only has intercepts for the random effects) or if we supress the correlations in the formula with `zerocorr()` then `θ` describes the relationship between the random effects standard deviation and the standard deviation of the residual term.
+In our `kb07_m` example:
+The residual standard deviation is `680.032`.
+The standard deviation of our first variance component *`item - (Intercept)`* is `364.713`.
+Thus our first `θ` is the relationship: variance component devided by residual standard deviation
+364.713 /  680.032 =  `0.53631`
+
+```@example Main
+kb07_m.θ
+```
+
+We also can calculate the `θ` for variance component *`subj - (Intercept)`*.
+The residual standard deviation is `680.032`.
+The standard deviation of our variance component *`subj - (Intercept)`* is `298.026`.
+Thus, the related θ is the relationship: variance component devided by residual standard deviation
+298.026 /  680.032 =  `0.438252`
+
+```@example Main
+kb07_m.θ
+```
+
+We can not calculate the `θ` for variance component *`item - prec: maintain`* yet, because it includes the correlation of
+*`item - prec: maintain`* and *`item - (Intercept)`*.
+The `θ` vector is the flattened version of the variance-covariance matrix - a lowertrinangular matrix.
+The on-diagonal elements are just the standard deviations (the `σ`'s), If all off-diagonal elements are zero, we can use our
+calculation above. The off-diagonal elements are covariances and correspond to the correlations (the `ρ`'s).
+If they are unequal to zero, as it is in our `kb07`-dataset, we cannot recreate the variance-covariance matrix having the model output.
+We just take it from the model we already fitted
+
+See the two inner values:
+```@example Main
+kb07_m.θ
+```
+
+TODO: NEED HELP HERE, IS THEIR ANY WAY TO DEFINE VARIANCE-COVARIANCE MATRIX YET ?
```

Comment:
Yes! The preview docs render, but not real ones for some reason, so check out:

https://repsychling.github.io/MixedModelsSim.jl/previews/PR28/simulation/

Reply:
Thats cool. and useful for the variances.
But I didn't understand yet, if I have any suggestions about the correlations (other than 1 and 0) if I can specify that. 
More precisely: is their a explainable relation between my suggestions and the resulting theta values?

In our example kb07: How can I calculate the theta values:
 
-0.25981993107379
 0.2653016002105174
 
representing 
Corr. -0.70 
"prec: maintain" Std.Dev. 252.521 , and 
knowing all the other model parameter.

Reply:
I've added some comments to this in the Markdown file